DataManagementSystem/Client/FTSRequest.py

```diff
@@ -1,6 +1,3 @@
-#############################################################################
```

Comment:
@chaen I re-changed this FTSRequest to what was before. This is of course buggy https://buildlhcb.cern.ch/jenkins/view/DIRAC/job/dirac-branch-v6r13/violations/file/DIRAC/DataManagementSystem/Client/FTSRequest.py/ so I let you solve it. 
@atsareg this can then be merged, but then Chris will make a PR to fix this